core/app/models/spree/option_value.rb

```diff
@@ -11,6 +11,14 @@ class OptionValue < Spree::Base
       validates :presentation
     end
 
+    scope :filterable, lambda {
+      joins(:option_type, :variants).
```

Comment:
@mad-eel why do we need to join variants here? we only need to join option type to apply this filter :)

Reply:
I wasn't sure at first, too 😄 But I meant for it to only return option values that have variants assigned.
It's not needed though and unclear so I'll remove it.